assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,32 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
```

Comment:
awesome job with this. I really wish this wasn't stretch, I feel everyone should go through this part of the assignment. Great job